tools/fvm-bench/src/fevm.rs

```diff
@@ -18,11 +18,21 @@ pub fn run(
     let create_res = testkit::fevm::create_contract(tester, &mut account, contract)?;
 
     if create_res.msg_receipt.exit_code.value() != 0 {
-        return Err(anyhow!(
-            "actor creation failed: {} -- {:?}",
-            create_res.msg_receipt.exit_code,
-            create_res.failure_info,
-        ));
+        println!("Execution trace:");
+        for tr in create_res.exec_trace {
+            println!("{:?}", tr)
+        }
+        return Err(match create_res.failure_info {
+            Some(fi) => anyhow!(
+                "contract creation failed: {} -- {}",
+                create_res.msg_receipt.exit_code,
+                fi
+            ),
+            None => anyhow!(
+                "contract creation failed: {}",
+                create_res.msg_receipt.exit_code
+            ),
+        });
```

Comment:
Why not keep this like it was?

Reply:
"{:?}" versus "{}". It was _debug_ printing backtraces (hard to read) instead of formatting them as strings (we have a nice and readable format).

Unfortunately, `Option` doesn't implement `Display`, hence this logic.